tools/setup.sh

```diff
@@ -0,0 +1,12 @@
+VERSION=0.6.1
+ARCH=$(uname -m)
+
+echo "=> Downloading HEMTT..."
+wget -q --show-progress https://github.com/synixebrett/HEMTT/releases/download/v$VERSION/hemtt-v$VERSION-$ARCH-unknown-linux-gnu.tar.gz
+
+echo "=> Extracting HEMTT..."
+tar -xzf hemtt-v$VERSION-$ARCH-unknown-linux-gnu.tar.gz -C ../.
+rm hemtt-v$VERSION-$ARCH-unknown-linux-gnu.tar.gz
+
+echo "=> Updating HEMTT..."
+../hemtt update
```

Comment:
Should it really auto update? Future changes in HEMTT might not work with the current project setup such as hemtt.toml.

Reply:
I am not sure if I'll keep those scripts around at all yet.

Reply:
A npm like update scheme might be good here? only update patches, but not minor?

Reply:
I think it's better to use a known working version and then update the HEMTT link once a newer version have been tested successfully.